docs/getting_started/quick_install.md

```diff
@@ -0,0 +1,74 @@
+# Quick install
+
+```{note}
+   These instructions assume familiarity with virtual environments and the
+   [Django web framework](https://www.djangoproject.com/).
+   For more detailed instructions, see [](tutorial).
+   To add Wagtail to an existing Django project, see [](integrating_into_django).
+```
+
+## Dependencies needed for installation
+
+-   [Python 3](https://www.python.org/downloads/).
+-   **libjpeg** and **zlib**, libraries required for Django's **Pillow** library.
+    See Pillow's [platform-specific installation instructions](https://pillow.readthedocs.io/en/stable/installation.html#external-libraries).
+
+## Install Wagtail
+Run the following commands in a virtual environment of your choice:
+
+```sh
+pip install wagtail
+```
+
+To install Wagtail outside a virtual environment, use the `sudo` command. `sudo` is a program that allows running other programs with the security privileges of a different user. This different user is by default the superuser.
+
+Once installed, Wagtail provides a command similar to Django's `django-admin startproject` to generate a new site/project:
```

Comment:
Should we add this comment? For those not familiar with Django it can just be noise, for those familiar with Django they will probably know this. 

If we do keep it, maybe we should add a link to the Django docs for this thing.